component-sets/material/examples/gallery/ui/views/navigation_view.slint

```diff
@@ -430,4 +446,15 @@ export component NavigationView {
             }
         ];
     }
+
+    time_picker := TimePicker {
+        x: -root.absolute_position.x;
+        y: -root.absolute_position.y;
+        width: MainWindowAdapter.width;
+        height: MainWindowAdapter.height;
```

Comment:
It's not entirely self-explaining to me what kind of placement is the result of these values. I suggest to add a comment (this is an example, after all).